addons/stock/tests/__init__.py

```diff
@@ -4,3 +4,4 @@
 import test_product
 import test_warehouse
 import test_shipment
+from . import test_stock_location_search
```

Comment:
What is your reference for this request?

Could you check the following pep https://www.python.org/dev/peps/pep-0328/#rationale-for-relative-imports?